### ISSUE TITLE
Upgrade to Node 8.12 to address CPU usage problems related to fibers

### ISSUE THREAD

**Issue by jimrandomh** [meteor/meteor#10117]
Node has a bug fix related to performance with fibers, which is very important to Meteor. Without the fix applied, any Meteor process which creates a large number of fibers will become very slow; switching between fibers is O(n) in the peak number of concurrent fibers the process has ever contained.

The fix is present in Node 10.2.0 and later, and which will be in Node 8.12 as soon as they cut a release, which should be soon. Meteor is currently on Node 8.11, which is affected.

Node 8.12 release PR page: https://github.com/nodejs/node/pull/21593

**Comment by harrisonhunter** [comment]
This should be handled in this PR: https://github.com/meteor/meteor/pull/10090 which looks like it should land in 1.7.1

**Comment by macrozone** [comment]
node 8.12 has been released! https://nodejs.org/en/blog/release/v8.12.0/

**Comment by benjamn** [comment]
You can start using Node 8.12.0 in the latest Meteor 1.7.1 release candidates!
```bash
meteor update --release 1.7.1-rc.11
```

**Comment by benjamn** [comment]
See also https://github.com/meteor/meteor/issues/10216, which is why we've reverted the upgrade for now.

**Comment by benjamn** [comment]
In order to reflect what's been fixed in the Meteor 1.8.1 milestone so far, I'm going to close this issue now, though of course you'll need to run
```bash
meteor update --release 1.8.1-beta.8
```
to get Node 8.14.0.
